platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformInputArtifactIntegrationTest.groovy

```diff
@@ -175,7 +196,7 @@ class ArtifactTransformInputArtifactIntegrationTest extends AbstractDependencyRe
         "@PathSensitive(PathSensitivity.NONE)"      | "@PathSensitive(PathSensitivity.NONE)"
     }
 
-    def "can attach #description to input artifact property with project artifact directory but it has no effect when not caching"() {
+    def "can attach #description to input artifact property with incrementally transformed artifact directory but it has no effect when not caching"() {
```

Comment:
❓ Do we now need some test coverage for non-incrementally transformed artifact directories as well?

Reply:
Added coverage for both incremental and non-incremental.